tests/rptest/utils/rpenv.py

```diff
@@ -23,3 +23,19 @@ def sample_license():
         assert is_ci == "false"
         return None
     return license
+
+
+class IsCIOrNotEmpty:
+    """
+    Comparison with this object is true if the environemnt is CI, or if the
+    other value is not empty.
+
+    Useful as a markup on a test that must run in CI or when the env is present:
+
+    @env(REQUIRED_ENV=IsCIOrNotEmpty())
+    """
+    def __init__(self):
+        self.is_ci = os.environ.get('CI', None) != 'false'
```

Comment:
If `CI` is not in the `os.environ` container, then `os.environ.get('CI', None)` returns `None` which means `None != 'false'` returns `true`

Should it by default return `'false'`?

Reply:
I just made it follow the pattern I previously mentioned, which is the one on Redpanda service.

Reply:
`sample_license()` above uses `get('CI', 'false')`

Reply:
```suggestion
        self.is_ci = os.environ.get('CI', 'false') != 'false'
```